options/ansi/generic/string-stubs.cpp

```diff
@@ -474,6 +491,132 @@ void *mempcpy(void *dest, const void *src, size_t len) {
 }
 
 // GNU extensions.
+const char *strerrorname_np(int e) {
+	const char *s;
+#define X(x) case x: s = #x; break;
+	switch(e) {
+	X(EAGAIN)
+	X(EACCES)
```

Comment:
Shouldn't this be indented one more?

Reply:
The case statements I replaced were on the same indentation level - but I can indent it a level more of desired